.github/workflows/check_charts.yml

```
@@ -0,0 +1,78 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+    paths:
+      # Only run test and docker publish if some code have changed
+      - 'deployment/k8s/charts/**'
+      - '.github/workflows/check_charts.yaml'
+      - 'pyproject.toml'
+  pull_request:
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      # - name: Check Version
+        # run: |
+          # current_version=$(grep '^version=' pyproject.toml | cut -f2 -d= | tr -d ' ' | tr -d '"')
+          # app_version=$(grep 'appVersion:' deployment/k8s/charts/Chart.yaml | cut -f2 -d: | tr -d ' ')
+          # if [[ "$current_version" != "$app_version" ]]; then
+            # echo "❌ current version from pyproject.toml ($current_version) and appVersion from Chart.yaml ($app_version) differs";
+            # exit 1;
+          # fi
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.9.2
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.7.0
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --chart-dirs deployment/k8s --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --chart-dirs deployment/k8s --target-branch ${{ github.event.repository.default_branch }}
+
+      - name: Build container
+        uses: docker/build-push-action@v6
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          # See https://github.com/developmentseed/titiler/discussions/387
+          platforms: linux/amd64
+          context: .
+          file: Dockerfile
+          push: false
+          tags: "titiler:dev"
```

let's use the same docker name as in the docker build
```suggestion
          tags: "titiler-openeo:dev"
```

---

Adjusted.